sdk/sdktypes/trigger.go

```diff
@@ -89,11 +88,20 @@ func (p Trigger) ToValues() map[string]Value {
 	return map[string]Value{
 		"name": NewStringValue(p.read().Name),
 		"data": kittehs.Must1(NewStructValue(
-			NewStringValue("event_data"),
+			NewStringValue("trigger_data"),
```

Comment:
not sure I understand what this is doing
should this be a constant somewhere ?

Reply:
it's just fake name. Probably cut-n-pasted from event code. So I changed the existing typo in code.